Docs/ProjectSpec.md

```diff
@@ -353,6 +353,7 @@ Run script build phases added via **prebuildScripts** or **postBuildScripts**. T
 - [ ] **outputFiles**: **[String]** - list of output files
 - [ ] **shell**: **String** - shell used for the script. Defaults to `/bin/sh`
 - [ ] **runOnlyWhenInstalling**: **Bool** - whether the script is only run when installing (`runOnlyForDeploymentPostprocessing`). Defaults to no
+- [ ] **showEnvVars**: **Bool** - whether the environment variables accessible to the script show be printed to the build log. Defaults to yes
```

Comment:
To mirror the dialog, should this be placed above `runOnlyWhenInstalling`?

Reply:
done